### PR TITLE
[13.0][FIX+IMP] sale_financial_risk: Compute risk for partner invoice. Inheritable method.

### PR DESCRIPTION
**Pull request by carlosdauden**
This PR solves https://github.com/OCA/credit-control/issues/84

@Tecnativa TT25596

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by ernestotejeda — APPROVED** [review]

File: sale_financial_risk/models/sale.py [hunk 1]
```diff
@@ -8,29 +8,34 @@
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    def evaluate_risk_message(self):
+        partner = self.partner_invoice_id.commercial_partner_id
```
Comment (pedrobaeza) [inline sale_financial_risk/models/sale.py]: As you are extracting that code, decouple it passing the partner as argument:
```suggestion
    def evaluate_risk_message(self, partner):
```

File: sale_financial_risk/models/sale.py [hunk 2]
```diff
@@ -8,29 +8,34 @@
...
-            elif partner.risk_sale_order_limit and (
-                (partner.risk_sale_order + risk_amount) > partner.risk_sale_order_limit
-            ):
-                exception_msg = _("This sale order exceeds the sales orders risk.\n")
-            elif partner.risk_sale_order_include and (
-                (partner.risk_total + risk_amount) > partner.credit_limit
-            ):
-                exception_msg = _("This sale order exceeds the financial risk.\n")
+            partner = self.partner_invoice_id.commercial_partner_id
+            exception_msg = self.evaluate_risk_message()
```
Comment (pedrobaeza) [inline sale_financial_risk/models/sale.py]: Following the other change:
```suggestion
            exception_msg = self.evaluate_risk_message(partner)
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

File: sale_financial_risk/models/sale.py (hunk 1 above)
Comment (carlosdauden) [reply sale_financial_risk/models/sale.py]: Ups, that was the idea...

**Pull request merged**